Xamarin.Forms.Platform.Tizen/Shell/INavigationView.cs

```diff
@@ -13,6 +12,8 @@ public interface INavigationView
 
 		EColor BackgroundColor { get; set; }
 
+		NImage BackgroundImage { get; set; }
```

Comment:
Couldn't we use ElmSharp.Image instead?

Reply:
Plus, if we can't implement this in the material renderer now, does it make sense to define it in the interface?

Reply:
That's just a limitation of `MNavigation`(`Tizen.NET.MaterialComponents`), It can be solved to re-implement `MaterialNavigationView` without `MNavigationView`, or to fix `MNavigationView`.

I think that it is better to fix `MNavigationView`, but it makes this PR delay until `Tizen.NET.MaterialComponents` package is released.

I'd like to suggest to make new PR for `FlyoutBackgroundImage` for Material Shell after `MNavigationView` is fixed.